scripts/20210706_remove-endpage-buttons/delete-endpage-buttons.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+
+/*
+Delete endPage.buttons key from old form documents in the database
+*/
+
+// == PRE-UPDATE CHECKS ==
+
+// Count total number of forms
+db.getCollection('forms').count() 
+
+// Count number of forms with endPage buttons key
+db.getCollection('forms').find({ "endPage.buttons": { $exists: true } }).count()
+
+
+// == UPDATE ==
+// Delete endPage buttons key
+// ~ Number of forms updated should match number which had endPage buttons key
+db.getCollection('forms').updateMany({}, {
+  $unset: {
+    'endPage.buttons': "",
+  }
+})
+
+// == POST-UPDATE CHECKS ==
+
+// Check total forms count
+// ~ Should be same as before
+db.getCollection('forms').count()
```

Comment:
I'm not sure there's value in counting the total number of forms. forms are constantly being created (even if we run this on a weekend night), so this is unlikely to be the same as the previous count anyway.